templates/greenlight/env.j2

```diff
@@ -127,6 +127,9 @@ ALLOW_MAIL_NOTIFICATIONS={{ bbb_allow_mail_notifications | ternary("true", "fals
 # Please note that enable this presents its own security risks and should not be done unless necessary.
 #   SMTP_OPENSSL_VERIFY_MODE=none
 #
+{% if bbb_greenlight_smtp.openssl_verify_none  %}
+SMTP_OPENSSL_VERIFY_MODE=none
```

Comment:
If my comment on the `README.md` change makes sense, then this needs to be changed as well.
But leaving it disabled by default unless its declared is the right way imo.